horace_core/sqw/file_io/@faccess_sqw_v4/get_exp_info.m

```diff
@@ -0,0 +1,46 @@
+function [exper,obj]  = get_exp_info(obj,varargin)
+% Get full data header or headers for sqw file written in format v3
+%
+% If instrument and sample are present in the file (not the empty
+% structures) it loads instruments and samples from the file and attaches
+% them to the header(s)
+%
+% Usage:
+%>>exp_info = obj.get_exp_info();       -- get header number 1
+%>>exp_info = obj.get_exp_info(1);      -- get header number 1
+%>>exp_info = obj.get_exp_info(number); -- get header with specified number
+%>>exp_info = obj.get_exp_info('-all');
```

Comment:
The API is historical and is not very convenient. It would be nice to change it but all previous loaders accept it as provided, so its a major job. The routine should return bunch of headers given array of numbers and fail on cellarray (this will certainly happen). The returning bunch of headers on array of request is not tested, 1 for 1 and all for `'-all'` do tested. 
I've added description of expected behavior on array of numbers but do no have time to test it properly (dirty PR)